wp-content/civi-extensions/goonjcustom/api/v3/Goonjcustom/FeedbackDroppingCenterCron.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @file
+ */
+
+use Civi\Api4\EckEntity;
+use Civi\Api4\OptionValue;
+use Civi\DroppingCenterFeedbackService;
+use Civi\HelperService;
+
+/**
+ * Goonjcustom.FeedbackDroppingCenterCron API specification (optional)
+ * This is used for documentation and validation.
+ *
+ * @param array $spec
+ *   description of fields supported by this API call.
+ *
+ * @return void
+ */
+function _civicrm_api3_goonjcustom_feedback_dropping_center_cron_spec(&$spec) {
+  // There are no parameters for the Goonjcustom cron.
+}
+
+/**
+ * Goonjcustom.DroppingCenterCron API.
+ *
+ * @param array $params
+ *
+ * @return array API result descriptor
+ *
+ * @see civicrm_api3_create_success
+ * @see civicrm_api3_create_error
+ *
+ * @throws \CRM_Core_Exception
+ */
+function civicrm_api3_goonjcustom_feedback_dropping_center_cron($params) {
+  $returnValues = [];
+
+  // Retrieve the Status option value.
+  $optionValues = OptionValue::get(FALSE)
+    ->addWhere('option_group_id:name', '=', 'eck_sub_types')
+    ->addWhere('name', '=', 'Status')
+    ->addWhere('grouping', '=', 'Dropping_Center_Meta')
+    ->setLimit(1)
```

Comment:
```suggestion
```
Should we need this here? If you are using `->single();` then we don't need `->setLimit(1)` please remove it and test the feature